operate/client/e2e-playwright/tests/processInstance.spec.ts

```diff
@@ -92,8 +88,11 @@ test.beforeAll(async ({request}) => {
 });
 
 test.describe('Process Instance', () => {
-  test('Resolve an incident', async ({page, processInstancePage}) => {
-    test.setTimeout(DEFAULT_TEST_TIMEOUT + 3 * 15000); // 15 seconds for each applied operation in this test
```

Comment:
I removed this in favor of `test.slow()`.

Before: 75s timeout (30s + 45s)
After: 90s timeout (30s * 3)